packages/oui-ui-select/ui-select.less

```diff
@@ -112,67 +114,114 @@
     }
   }
 
-  .ui-select-match {
-    #oui > .button-normalize();
-    #oui > .button-base();
-    #oui > .button-hollow();
-    #oui > .button-dropdown();
+  /* Helpers */
+  .ui-select-highlight { font-weight: @oui-font-bold; }
 
-    max-width: 100%;
-    width: 100%;
-    height: @oui-ui-select-height;
-    text-align: left;
+  .ui-select-offscreen {
+    clip: rect(0 0 0 0);
+    width: 1px;
+    height: 1px;
+    border: 0;
+    margin: 0;
+    padding: 0;
+    overflow: hidden;
+    position: absolute;
+    outline: 0;
+    left: 0;
+    top: 0;
+  }
 
-    &:focus,
-    &_focus {
-      background-color: #f2f2f2;
-      border-color: #999;
+  /* Variant 'multiple' */
+  &_multiple {
+    .ui-select-match,
+    .ui-select-match-items {
+      line-height: @oui-ui-select-multiple-line-height;
+    }
 
-      /* To override reset style */
-      &[tabindex="-1"] {
+    .ui-select-match-container {
+      #oui > .button-normalize();
+      #oui > .button-base();
+      #oui > .button-hollow();
+      #oui > .button-dropdown(@icon-selector: oui-icon-chevron-down);
+
+      max-width: 100%;
+      width: 100%;
+      min-height: @oui-ui-select-height;
+      line-height: @oui-ui-select-multiple-line-height;
+      padding-top: @oui-ui-select-multiple-padding-y;
+      padding-bottom: @oui-ui-select-multiple-padding-y;
+      text-align: left;
+      white-space: initial;
+
+      &:focus,
+      &_focus {
+        background-color: #f2f2f2;
```

Comment:
```suggestion
        background-color: @oui-color-concrete;
```

Reply:
Good catch, I'll update all in one commit.